setup.py

```diff
@@ -199,6 +201,16 @@ def run(self):
         'clean': clean
     },
     'entry_points': {
+        'kiwi.tasks': [
```

Comment:
I guess having them all listed here is the counter part of the refactor of cli.py based on `pkg_resources`. I did not expect that, but I understand this is better than having to parse the OS file within `kiwi/tasks` folder.

Reply:
yeah let's keep this part explicit. I actually also like it better, it's not a real hurdle